Decred Wallet/Features/Receive/ReceiveViewController.swift

```diff
@@ -0,0 +1,221 @@
+//  ReceiveViewController.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import Foundation
+import UIKit
+import Dcrlibwallet
+
+class ReceiveViewController: UIViewController, UIDocumentInteractionControllerDelegate {
+    @IBOutlet weak var menuBtn: UIButton!
+    @IBOutlet private var imgWalletAddrQRCode: UIImageView!
+
+    @IBOutlet weak var walletAddressLabel: UILabel!
+    @IBOutlet weak var containerRoundedView: RoundedView!
```

Comment:
```suggestion
    @IBOutlet weak var mainContentViewHolder: RoundedView!
```

This should be positioned after the `moreMenuButton` declaration, before the contained views.